packages/core/src/motify.tsx

```diff
@@ -21,12 +16,8 @@ export default function motify<
     // outerProps?: ExtraProps
     {
       const withStyles = forwardRef<
-        Ref,
-        Props &
-          MotiProps<Animate> &
-          ExtraProps & {
-            children?: React.ReactNode
-          }
+        ComponentType<Props>,
```

Comment:
Is this the correct way to type `forwardRef`'s ref?

Reply:
Yeah, maybe we could look to other libs who forward refs to see what they do with types. I'd also like to know what to do here for Dripsy. Maybe try the existing ref type and see if it's properly typed?

Something like:

```jsx 
const ref = useRef<Text>(null)

return <MotiText ref={ref} />
```

Where `Text` is imported from React Native directly, and see if that works.

Reply:
I believe I've got a fix in place that will work with the following example:

```tsx
const MView = motify(View)()
const MText = motify(Text)()

export const FooBar: React.FC = () => {
  const ref = React.useRef<View>(null)

  return (
    <MView animate={{ translateX: 3 }} ref={ref}>
      <MText animate={{ color: 'red' }} />
    </MView>
  )
}
```

Just pushed up a commit with the change. Let me know what you think. Basically, just changing `ComponentType<Props>` to `Component<Props>` for that first generic for `forwardRef` seems to do the trick.